src/components/post/postTemplate.js

```diff
@@ -67,8 +67,7 @@ export const pageQuery = graphql`
       html
       frontmatter {
         date
-				title
-				subtitle
+		title
```

Comment:
You removed the `subtitle` query here, which is needed for rendering posts.
You should keep it.